ores/wsgi/templates/home.html

```diff
@@ -42,6 +42,20 @@ <h1 class="site__title"><a href="./"><span class="site__logo"></span>ORES</a></h
   <div class="page">
     <div class="content-box">
       <main id="content" class="content" role="main">
+        <div class="mw-style-warning-alert" style="padding-top: 15px">
+            <div class="header">
+                Deprecation Warning:
+            </div>
+            <p>
+                The ORES infrastructure is being deprecated. You can continue scoring revisions using the same models which are now available on Lift Wing.
```

Comment:
I know that the ORES http link below should be enough, but I'd also add another a href around "Lift Wing" pointing it to https://wikitech.wikimedia.org/wiki/Machine_Learning/LiftWing/Usage. Same thing in scorer.html

Reply:
Thanks @elukey, a link to the LiftWing usage has been added.